### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by elenapeskova**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,24 +1,55 @@
...
-* [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
-* [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
-<!-- default file list end -->
-# ASPxGridView - How to apply a filter to a column on the client side
+# GridView for ASP.NET Web Forms - How to apply a filter to a column on the client side
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e3583/)**
 <!-- run online end -->
 
+This example demonstrates how to use a text string from the external text box to filter a grid column on the client.
```
Comment (dmitry-eliseev-devexpress) [inline Readme.md]: > This example demonstrates how to use a text string from **the** external text box to filter a grid column on the client.

Of what external text box do we speak? We do not know anything of it just yet. This entire introductory sentence speaks of general and abstract things, like it should. Therefore, we should not make an exception of this external text box and make it indefinite.

```suggestion
This example demonstrates how to use a text string from an external text box to filter a grid column on the client.
```

File: Readme.md [hunk 2]
```diff
@@ -1,24 +1,55 @@
...
+
+![AutoFilterByColumn](AutoFilterByColumn.gif)
```
Comment (dmitry-eliseev-devexpress) [inline Readme.md:8]: While the animation itself is good, I can't help but notice a few issues related to the source application itself:

1. Your filter button is not aligned with anything -- neither with the search box, nor with your grid. Moreover, the button visually "stuck" to the grid. This does not look professional. I believe you can adjust your application layout better and re-capture the animation.
2. The button caption says "Filter by **CategoryName**". Why did you chose the form "CategoryName" over "Category Name"? The column's caption says "Category Name" rather than "CategoryName", and all UI names should have a direct connection to the UI entities they are referring to. I recommend you to rename this button "Filter by Category Name".

**Bottom Line** -- I recommend you to re-capture this animation taking the mentioned issues into consideration. Thanks in advance.

File: Readme.md [hunk 3]
```diff
@@ -1,24 +1,55 @@
...
+            <dx:GridViewDataTextColumn FieldName="CategoryName" VisibleIndex="1" />
+            <dx:GridViewDataTextColumn FieldName="Description" VisibleIndex="2" />
+        </Columns>
+    </dx:ASPxGridView>
+    <asp:AccessDataSource ID="ds" runat="server" DataFile="~/App_Data/nwind.mdb"
+        SelectCommand="SELECT [CategoryID], [CategoryName], [Description] FROM [Categories]">
+    </asp:AccessDataSource>
+    ```
+
+2. Add a text box and button. When a user enters a text string in the text box and clicks the button, the grid filters a particular column by the entered text. To enable this behavior, handle the button's client-side [Click](https://docs.devexpress.com/AspNet/js-ASPxClientButton.Click) event. In the handler, call the grid's [AutoFilterByColumn](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.AutoFilterByColumn(column-val)) method and use the text from the text box as a parameter.
```
Comment (dmitry-eliseev-devexpress) [inline Readme.md]: > Add a text box and button.

You urge a reader to add **one** text box and **one** button. In this context, "text box" and "button" are two different entities that cannot be joined under the same article.

> When a user enters a text string in the text box and clicks the button, the grid filters a particular column by the entered **text**.

The sentence starts good and well, however, you use "text" as synonym for "text string" in the end. Again, I always stand against overuse of the word "text" as a noun. And this sentence introduces an implied synonym. Instead, you should have used the same noun as in the beginning of your sentence to have an unbroken connection.

> use the **text** from the text box as a parameter.

Likewise, this is not good. "Text" as a noun is a very abstract thing -- it can be literally anything from a sentence to fundamental works of words -- such as books, treatises, religious hymns, or poems. When you speak of parameters, you go down to technical down-to-earth things, therefore, you should use suitable words. In technical sense, you use the current edit value of the text box as a parameter.

Finally, when you speak of parameters, you **pass** a value to your method as a parameter. Yes, you could use "use" here, however, if we can use a more precise word in this context, we should.

Here is my current take on this paragraph:

```suggestion
2. Add a text box and a button. When a user enters a text string in the text box and clicks the button, the grid filters a particular column by the entered string. To enable this behavior, handle the button's client-side [Click](https://docs.devexpress.com/AspNet/js-ASPxClientButton.Click) event. In the handler, call the grid's [AutoFilterByColumn](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.AutoFilterByColumn(column-val)) method and pass the edit value of the text box as a parameter.
```

**Review by dmitry-eliseev-devexpress — COMMENTED** [review]
**Structure and Semantics**

This is a small topic that explains how to implement a simple scenario. The topic serves its intended purpose in its current state and has no structural and semantic issues. Well done.

**Images**

While the animation itself is clean and clear, the source application has a couple of UI issues I described in the corresponding comment. I strongly recommend you to rectify these issues and re-capture the animation.

**Language**

Make no mistake -- I admire your efforts in making your language better and I see the results. The issues I pointed out do not prevent one to understand the topic properly. But you can't have too much precision and clarity in your text, especially when it comes to documentation.

The main language focus of this review: please do be mindful of words with broad meaning -- such as the verb "to use" and the noun "text". Especially with "text" because it can be literally anything and we don't want that, do we?

**Bottom Line**

Good job. But I'll approve this topic once I see a new version of your animation. Thanks in advance.

**Review by dmitry-eliseev-devexpress — APPROVED** [review]
Everything is looking fine in the updated GIF animation. Good to go!

File: Readme.md (hunk 2 above)
Comment (dmitry-eliseev-devexpress) [reply Readme.md:8]: The issue is resolved.
